schema/subnetGroupsSchema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$id": "/schemas/SubnetGroups.Schema.json",
+  "description": "Schema for the subnet groups file",
+  "additionalProperties": false,
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "subnetGroupName": {
+        "type": "string"
```

Comment:
Would recommend we put some kind of requirements on the subnet group name.  Starting simple and small would be preferable.

Reply:
Yep agreed.  Adding maxLength